### PR TITLE
Allow extensions on the root element

### PR DESCRIPTION
**Pull request by lnicola**
- [x] I agree to follow the project's [code of conduct](https://github.com/georust/geo/blob/master/CODE_OF_CONDUCT.md).
- [x] I added an entry to `CHANGELOG.md` if knowledge of this change could be valuable to users.
---

Fixes #90

### CODE REVIEW

File: src/parser/gpx.rs [hunk 1]
```diff
@@ -224,6 +229,9 @@ mod tests {
                     <time>2001-10-26T19:32:52+00:00</time>
                 </wpt>
                 <rte></rte>
+                <ql:extensions>
```
Comment (supersnellehenk) [inline src/parser/gpx.rs:232]: `<ql:extensions>` should be `<extensions>` as the sample file in #89 ?

File: src/parser/mod.rs [hunk 2]
```diff
@@ -5,8 +5,8 @@
 #[macro_export]
 macro_rules! consume {
     ($xml:expr, $version:expr) => {{
-        use $crate::parser::create_context;
         use std::io::BufReader;
+        use $crate::parser::create_context;
```
Comment (lnicola) [inline src/parser/mod.rs:9]: `cargo fmt` did this.

**Comment by urschrei** [comment]
This is https://www.topografix.com/GPX/1/1/#type_extensionsType, right? It's so vague.

**Comment by lnicola** [comment]
The file is.. GPX 1.0?

```
<gpx xmlns:gpxdata="http://www.cluetrust.com/XML/GPXDATA/1/0" xmlns:gpxx="http://www.garmin.com/xmlschemas/GpxExtensions/v3" version="1.1" xmlns="http://www.topografix.com/GPX/1/1" xmlns:rmc="urn:net:trekbuddy:1.0:nmea:rmc" xmlns:gpxtpx="http://www.garmin.com/xmlschemas/TrackPointExtension/v1" xsi:schemaLocation="http://www.topografix.com/GPX/1/1 http://www.topografix.com/GPX/1/1/gpx.xsd http://www.garmin.com/xmlschemas/GpxExtensions/v3 http://www.garmin.com/xmlschemas/GpxExtensionsv3.xsd http://www.garmin.com/xmlschemas/TrackPointExtension/v1 http://www.garmin.com/xmlschemas/TrackPointExtensionv1.xsd http://www.garmin.com/xmlschemas/WaypointExtension/v1 http://www.garmin.com/xmlschemas/WaypointExtensionv1.xsd http://www.qlandkarte.org/xmlschemas/v1.1 http://www.qlandkarte.org/xmlschemas/v1.1/ql-extensions.xsd http://www.cluetrust.com/XML/GPXDATA/1/0 http://www.cluetrust.com/Schemas/gpxdata10.xsd" xmlns:wptx1="http://www.garmin.com/xmlschemas/WaypointExtension/v1" xmlns:ql="http://www.qlandkarte.org/xmlschemas/v1.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" creator="QMapShack 1.16.1 http://www.qlandkarte.org/" xmlns:tp1="http://www.garmin.com/xmlschemas/TrackPointExtension/v1">
```

**Comment by lnicola** [comment]
https://www.topografix.com/GPX/1/1/#element_gpx shows it more clearly.

**Review by urschrei — APPROVED** [review]

**Comment by lnicola** [comment]
bors r=urschrei

**Pull request merged**